src/pages/kiman/Main/Main.js

```diff
@@ -1,9 +1,87 @@
-import React from 'react';
-import './Main';
-import Nav from '../../../components/Nav/Nav';
+import './Main.scss';
+import '../../../../src/common.scss';
+import '../../../../src/reset.scss';
+import { useState, useEffect } from 'react';
+
+import Nav from '../../../../src/components/Nav/Nav';
+import MainFeed from './MainFeed/MainFeed';
+import StoryData from './MainFeed/Othersdata/StoryData';
+import PushData from './MainFeed/Othersdata/PushData';
 
 const Main = () => {
-  return <Nav />;
+  const [mockDatas, setMockDatas] = useState('');
+
+  useEffect(() => {
+    fetch('http://localhost:3000/data/commentData-kiman.json')
```

Comment:
`http://localhost:3000` 은 생략 가능. 포트 번호가 달라질 경우(ex. localhost:3001) 에러가 발생하고, 코드 전체를 바꿔야 하기 때문에 생략하는 게 좋음
```suggestion
    fetch('/data/commentData-kiman.json')
```